src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
-class Header {
+const f = 'Conduit logo';
+const y = '[Publish comment] button';
```

Comment:
There is no need to create variables because it's hard to read what your code does or what it returns. 
For instance,  reading `return 'Click on the ' + this.commentButton;` is much easier than ` return 'Click on the ' + y;`

Reply:
agree